src/PatientTestsApi/LoadPatient/function.json

```diff
@@ -16,5 +16,5 @@
       "name": "res"
     }
   ],
-  "scriptFile": "../dist/LoadPatient/index.js"
+  "scriptFile": "../dist/UpdatePatient/index.js"
```

Comment:
I think you modified the wrong file here. Shouldn't it be the UpdatePatient/function.json ?